### PR TITLE
feat: show created at and update at times

### LINKED ISSUE #210: Show createdAt and updatedAt

### PR DESCRIPTION
**Pull request by clementcharmillot**
image

closes #210

### CODE REVIEW

File: src/components/collection/Summary.js [hunk 1]
```diff
@@ -240,6 +242,27 @@ function Summary({
...
+            <div>
+              <Typography variant="h6" gutterBottom>
+                Created At
+              </Typography>
+              <Typography variant="p" gutterBottom>
+                {new Date(createdAt).toLocaleDateString()}
+              </Typography>
+            </div>
+          )}
+          {createdAt && (
```
Comment (spaenleh) [inline src/components/collection/Summary.js:255]:
```suggestion
          {lastUpdate && (
```

**Review by spaenleh — CHANGES_REQUESTED** [review]
🕙 Thank you for this PR ! I think you just have a small copy-paste issue.

A nice touch for PRs that improve/change UI elements is to include a screenshot of the result in the PR comment, so reviewers can see what your change looks like without needing to run the code 😊

In case you want to make further changes, feel free to re-request a review (♺ icon) !

**Review by spaenleh — APPROVED** [review]
Awesome ! Thank you for addressing my comments 💪

File: src/components/collection/Summary.js [hunk 2]
```diff
@@ -240,6 +244,27 @@ function Summary({
             )}
           </Typography>
           <Authorship itemId={itemId} author={creator} isLoading={isLoading} />
+          {createdAt && (
+            <div id={SUMMARY_CREATED_AT_CONTAINER_ID}>
+              <Typography variant="h6" gutterBottom>
+                Created At
+              </Typography>
+              <Typography variant="p" gutterBottom>
+                {new Date(createdAt).toLocaleDateString()}
```
Comment (pyphilia) [inline src/components/collection/Summary.js:253]: I would use maybe a nice library to have a nicer display, like `date-fns` or any recommendation from @dialexo

**Review by pyphilia — APPROVED** [review]

File: src/components/collection/Summary.js (hunk 2 above)
Comment (dialexo) [reply src/components/collection/Summary.js:253]: Very good point, i would suggest luxon, the immutable successor to moment: https://moment.github.io/luxon/

**Review by dialexo — CHANGES_REQUESTED** [review]
Thank you for your work! See my comments, otherwise LGTM and we can merge 👍

**Review by dialexo — APPROVED** [review]

**Pull request merged**